### PR TITLE
DBZ-1292 ☁️ Initial import of CloudEvents example

### PR DESCRIPTION
**Pull request by gunnarmorling**
Example for CloudEvents support https://issues.redhat.com/browse/DBZ-1292

Requires https://github.com/debezium/debezium/pull/1198.

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@Naros, any thoughts on this one?

File: cloudevents/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Debezium CloudEvents Demo
+
+This demo automatically deploys the topology of services as defined in the [Debezium Tutorial](http://debezium.io/docs/tutorial/).
+
+## Preparations
+
+```shell
+export DEBEZIUM_VERSION=1.0
```
Comment (Naros) [inline cloudevents/README.md]:
```suggestion
export DEBEZIUM_VERSION=1.1
```

File: cloudevents/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# Debezium CloudEvents Demo
+
+This demo automatically deploys the topology of services as defined in the [Debezium Tutorial](http://debezium.io/docs/tutorial/).
+
+## Preparations
+
+```shell
```
Comment (Naros) [inline cloudevents/README.md]: Should we add a step to run `mvn install` for avro-data-converter?

File: cloudevents/avro-data-extractor/pom.xml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+
+        <mvn.compiler.version>3.5.1</mvn.compiler.version>
+        <mvn.dependency.version>3.0.2</mvn.dependency.version>
+        <version.surefire>2.22.0</version.surefire>
+
+        <apache.kafka.version>2.0.0</apache.kafka.version>
+        <version.quarkus>1.1.1.Final</version.quarkus>
+        <version.debezium>1.0.0.Final</version.debezium>
```
Comment (Naros) [inline cloudevents/avro-data-extractor/pom.xml]: Bump this to 1.1.0.Alpha1?

File: cloudevents/README.md (hunk 2 above)
Comment (gunnarmorling) [reply cloudevents/README.md]: Yes, good point.

File: cloudevents/avro-data-extractor/pom.xml (hunk 3 above)
Comment (gunnarmorling) [reply cloudevents/avro-data-extractor/pom.xml]: Yes, good point.

**Comment by gunnarmorling** [comment]
Addressed review remarks and merged. Thanks for reviewing, @Naros!

**Pull request merged**